src/CQRS/LeanCode.CQRS.AspNetCore/Registration/CQRSObjectsRegistrationSource.cs

```diff
@@ -10,17 +10,32 @@ internal class CQRSObjectsRegistrationSource
     private readonly TypesCatalog contractsCatalog;
     private readonly TypesCatalog handlersCatalog;
 
-    public IEnumerable<CQRSObjectMetadata> Objects { get; }
+    public IEnumerable<CQRSObjectMetadata> Objects { get; private set; }
 
     public CQRSObjectsRegistrationSource(TypesCatalog contractsCatalog, TypesCatalog handlersCatalog)
     {
         this.contractsCatalog = contractsCatalog;
         this.handlersCatalog = handlersCatalog;
 
-        Objects = FindCQRSObjects();
+        Objects = FindCQRSObjects(contractsCatalog, handlersCatalog);
     }
 
-    private IEnumerable<CQRSObjectMetadata> FindCQRSObjects()
+    public IEnumerable<CQRSObjectMetadata> AddAdditionalCQRSObjects(
```

Comment:
Well, at this point I think it's better to remove constructor parameters and have only one method `AddCQRSObjects(TypesCatalog, TypesCatalog)`.

By the way, what would happen now if you add the same assembly twice - I think it's gonna crash at some point (probably routing).